connectors/sources/salesforce.py

```diff
@@ -1502,92 +1527,99 @@ async def get_docs(self, filtering=None):
                     )
 
         else:
-            for sobject in [
-                "Account",
-                "Opportunity",
-                "Contact",
-                "Lead",
-                "Campaign",
-                "Case",
-            ]:
+            for sobject in objects_to_sync:
```

Comment:
I think it'd be nice to make this a hash instead and avoid `for` loop, it just makes it harder to perceive, e.g.:

```
objects_to_sync = { 'Account': 1, 'Opportunity': 0}

if 'Account' in objects_to_sync:
    ...

if 'Case' in objects_to_sync:
   ...
```